### PR TITLE
fix(patcher.js): rollup does not build dynamic imports

### PR DESCRIPTION
**Pull request by ranrib**
rollup does not compile dynamic imports (such as tryRequire) in the build.
It means that events/pg was never instrumented.

### CODE REVIEW

File: src/events/pg.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 const uuid4 = require('uuid4');
 const shimmer = require('shimmer');
+const tryRequire = require('try-require');
 /* eslint-disable import/no-unresolved, import/no-extraneous-dependencies */
-const pg = require('pg');
-const Pool = require('pg-pool');
-const sqlParser = require('node-sqlparser');
+const pg = tryRequire('pg');
+const Pool = tryRequire('pg-pool');
+const sqlParser = tryRequire('node-sqlparser');
```
Comment (galbash) [inline src/events/pg.js]: This should be a regular require

**Review by galbash — COMMENTED** [review]
1 small comment

File: src/events/pg.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 const shimmer = require('shimmer');
+const tryRequire = require('try-require');
 /* eslint-disable import/no-unresolved, import/no-extraneous-dependencies */
-const pg = require('pg');
-const Pool = require('pg-pool');
-const sqlParser = require('node-sqlparser');
+const pg = tryRequire('pg');
+const Pool = tryRequire('pg-pool');
 /* eslint-enable import/no-unresolved, import/no-extraneous-dependencies */
+const sqlParser = require('node-sqlparser');
```
Comment (galbash) [inline src/events/pg.js:4]: Remove the eslint line above

Comment (ranrib) [reply src/events/pg.js]: The eslint line above is `enable` (comes after the `disable`)

Comment (ranrib) [reply src/events/pg.js:4]: 👍

**Review by galbash — APPROVED** [review]

**Pull request merged**